php/regions/class-region.php

```diff
@@ -113,25 +130,26 @@ public function get_imposed_data() {
 
 	/**
 	 * Recursively get the current data for the region
-	 * 
+	 *
+	 * @param array $schema Schema array.
+	 *
```

Comment:
extremely minor, feel free to ignore... elsewhere in this PR there's no blank line between @param and @return lines.